cmake/Modules/FindERTPython.cmake

```diff
@@ -23,17 +23,17 @@ if(PYTHONINTERP_FOUND)
   endif()
   set(PYTHON_INSTALL_PREFIX  "lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/${PYTHON_PACKAGE_PATH}"  CACHE STRING "Subdirectory to install Python modules in")
 
+  list(APPEND PATH_LIST "")
```

Comment:
yes. all variables in cmake can be "morphed" into lists simply by using list(APPEND). so you just want to make sure that the variable starts out blank.